src/explore-education-statistics-common/src/modules/table-tool/utils/__tests__/optimizeFilters.test.ts

```diff
@@ -0,0 +1,41 @@
+import optimizeFilters from '../optimizeFilters';
+import {
+  testFilters1,
+  testHeaderConfig1,
+  testCategoryFilter1,
+  testFilters2,
+  testHeaderConfig2,
+  testCategoryFilter4,
+  testFilters3,
+  testHeaderConfig3,
+  testCategoryFilter8,
+  testFilters4,
+  testHeaderConfig4,
+  testLocationFilter1,
+} from '../__data__/testFilterData';
+
+describe('optimizeFilters', () => {
```

Comment:
It doesn't seem like any of the test cases currently make sense. Maybe it'd be best to jump on a call if you need help understanding what this function is supposed to do?

Reply:
I've changed all the test cases for optimizeFilters.